### ISSUE TITLE
Pass the absolute path of the resources directory to libservo

### ISSUE THREAD

**Issue by paulrouget** [paulrouget/servofocus#2]
I need the absolute path of the resources directory, where we put the Servo resources files.

No need to pass it to init just yet, I just need to know how to get it, as a `char *`.

/cc @mfkl

**Comment by paulrouget** [comment]
fwiw, resources can be copied from here: https://github.com/servo/servo/tree/master/resources

**Comment by paulrouget** [comment]
That's how init is called now:

```rust
pub extern "C" fn init(
    wakeup: extern fn(),
    resources_path: *const c_char
    width: u32
    height: u32
}
```

**Comment by mfkl** [comment]
Yeah. It's already in the repo https://github.com/paulrouget/servofocus/tree/master/xamarin/resources
Found a way to include them in the shared project (referenced by all apps) as Embed Resources. Now looking into how to find the correct path on each platform.

**Comment by paulrouget** [comment]
I don't know if that helps, but usually we use externalDataPath / internalDataPath to access these resources on Android.

**Comment by paulrouget** [comment]
So yeah, it's not possible to access resources with an absolute path. So a workaround is to copy the resources to the sdcard at startup. That sucks.

For now, we can just copy the resources folder manually, but a long term solution is for Servo to not use an absolute path but a stream.

**Comment by paulrouget** [comment]
I posted on dev-servo about this issue.

**Comment by mfkl** [comment]
Yeah I got the workaround implementation somewhere in my stash. Still want it?

**Comment by paulrouget** [comment]
Nope. Thanks. Let's do that manually for now.
